cibyl/models/ci/job.py

```diff
@@ -48,7 +48,10 @@ class Job(Model):
                                    description="Job builds"),
                           Argument(name='--last-build', arg_type=str,
                                    func='get_last_build', nargs=0,
-                                   description="Last build for job")]
+                                   description="Last build for job"),
+                          Argument(name='--build-status', arg_type=str,
```

Comment:
There is an identical Argument already in Build, is there any reason to add it here?

Reply:
+1 to what Joan wrote. It's a build attribute, so it fits better in the Build model.

Reply:
It makes sense, thank you Joan and Arie. In this case I should move the logic to the `get_builds` method. I'll do the change.

Reply:
Changed :+1: